src/main/java/org/mule/module/apikit/validation/attributes/QueryParameterValidator.java

```diff
@@ -51,8 +51,10 @@ private static ValidatedQueryParams validateQueryParams(Map<String, Parameter> q
       String paramKey = paramDefinitionEntry.getKey();
       List<String> values = incomingQueryParams.getAll(paramKey);
 
-      if (!values.isEmpty()) {
-        if (parameterDefinition.isRepeat() || parameterDefinition.isArray()) {
+      if (values != null && !values.isEmpty()) {
+        if (isSingleValueListOfNull(values) && parameterDefinition.isNullable()) {
```

Comment:
`Parameter.isNullable()` is not defined (the build is currently failing because of this). What happened? D: